src/styles/utils/variables.scss

```diff
@@ -0,0 +1,14 @@
+$day-width: 100px;
+$day-hight: 100px;
+$days-gap: 1px;
+$calendar-width: 7 * ($day-width + $days-gap);
+
+$days-name: (
+  mon:1,
+  tue:2,
+  wed:3,
+  thu:4,
+  fri:5,
+  sat:6,
+  sun:7,
+);
```

Comment:
I think that it is not a mistake but in video was another one, but as I told I don't sure that this is mistake)